membership_extension/views/product_template_view.xml

```diff
@@ -19,8 +19,8 @@
     <field name="model">product.template</field>
     <field name="inherit_id" ref="membership.membership_products_form" />
     <field name="arch" type="xml">
-        <field name="membership" position="after">
-            <field name="membership_category_id" />
+        <field name="company_id" position="after">
```

Comment:
Why changing the position?

Reply:
Because of this onchange: https://github.com/OCA/vertical-association/pull/81/files#diff-7e6c873c7ba99b0bf6d9e8391c33a96fR26-R28

If we keep `membership_category_id` before `company_id`, it results in a poor UX: imagine when the user sets a `membership_category_id, then sets the company and the onchange overwrites the previously selected `membership_category_id`.
To avoid such confusing behavior, I moved the field in a more logical position.

Reply:
Ok, but the two fields are together and `membership` is invisible anyway :sweat_smile: